<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramWindow</class>
 <widget class="QDialog" name="ProgramWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="input_filter_computer">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>100</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Search computer</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="Scientist_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>531</width>
     <height>411</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sex</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Year Born</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Year Died</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="Computer_table">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>140</y>
     <width>441</width>
     <height>411</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Year Built</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="add_computer_button">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>200</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Computer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_filter_scientist">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>531</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Search scientist</string>
   </property>
  </widget>
  <widget class="QLabel" name="Scientist_text_2">
   <property name="geometry">
    <rect>
     <x>681</x>
     <y>31</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;h2&gt;Computers&lt;/h2&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_scientist_button">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>250</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Scientist</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Scientist_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;h2&gt;Scientists&lt;/h2&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_remove_scientist">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>300</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Scientist</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_remove_computer">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>350</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Computer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
